automower_ble/protocol.json

```
@@ -184,7 +184,8 @@
         "major": 4690,
         "minor": 5,
         "requestType":{
-            "task": "uint8"
+            "unknown": "uint16",
+            "task": "uint16"
```

```suggestion
            "task": "uint32"
```
According to Automower apk this is a uint32. As you append `00 00` by setting `unknown` this code still works as you're essentially converting a uint16.